separate_curves.py

```diff
@@ -34,9 +34,9 @@
 from bpy.types import Operator
 
 
-class SeparateCurveOperator(Operator):
+class SCO_OT_operator(bpy.types.Operator):
```

Comment:
It's not a blocker, but the `Operator` is imported from `bpy.types`, so you can use here just `Operator`